### PR TITLE
Sign up modal

### LINKED ISSUE #78: `Sign up modal`
Screen Shot 2022-09-11 at 12 03 19 AM

### PR DESCRIPTION
**Pull request by byronwang93**
Closes #78

### CODE REVIEW

File: src/components/modal/signupmodal/SignupModal.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+const SubscribeButton = styled.button`
+  ${SubscribeButtonCSS}
+`;
+
+const SignupModalContainer = ({ clickEvent }) => {
+  return (
+    <SignupModalComponent>
+      <LogoImage src={Logo} />
+      <CloseIcon src={ClosedIcon} />
```
Comment (ajo01) [inline src/components/modal/signupmodal/SignupModal.js:53]: Make sure to add an onClick on the close icon as well. Maybe name it onCloseClick?

File: src/components/modal/signupmodal/SignupModal.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      <HeaderText>nwPlus Newsletter Sign-up</HeaderText>
+      <BodyText>
+        Subscribe to our newsletter to stay up to date and for upcoming events!
+      </BodyText>
+
+      <InputContainer>
+        <Input placeholder="hacker@nwplus.io" />
+        <SubscribeButton onClick={clickEvent}>Subscribe</SubscribeButton>
```
Comment (ajo01) [inline src/components/modal/signupmodal/SignupModal.js:61]: Would rename clickEvent to onSubscribeClick to follow standard react naming conventions

**Review by ajo01 — CHANGES_REQUESTED** [review]
Otherwise looks good!

**Comment by byronwang93** [comment]
@ajo01 addressed both points in recent commit ^

**Review by ajo01 — APPROVED** [review]

**Pull request merged**